install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+DB_USER="dicedb"
+DB_GROUP="dicedb"
+DB_CONF_DIR="/etc/dicedb"
+DB_CONF_FILE="$DB_CONF_DIR/dice.toml"
+DB_BIN="/usr/local/bin/dicedb"
+DB_SERVICE_FILE="/etc/systemd/system/dicedb.service"
+
+# Create the user and group if they don't exist
+if ! id -u $DB_USER >/dev/null 2>&1; then
+    echo "Creating user and group: $DB_USER"
+    sudo groupadd $DB_GROUP
+    sudo useradd -g $DB_GROUP -s /bin/false -d /nonexistent $DB_USER
+else
+    echo "User and group $DB_USER already exist"
+fi
+
+# Create configuration directory and file
+if [ ! -d "$DB_CONF_DIR" ]; then
+    echo "Creating configuration directory: $DB_CONF_DIR"
+    sudo mkdir -p $DB_CONF_DIR
+fi
+
+if [ ! -f "$DB_CONF_FILE" ]; then
+    echo "Creating configuration file: $DB_CONF_FILE"
+    sudo touch $DB_CONF_FILE
+    sudo chown root:$DB_GROUP $DB_CONF_FILE
+    sudo chmod 640 $DB_CONF_FILE
+    # curl "https://raw.githubusercontent.com/DiceDB/dice/master/dice.toml" | sudo tee $DB_CONF_FILE
+    cp ./dice.toml $DB_CONF_FILE
+else
+    echo "Configuration file $DB_CONF_FILE already exists"
+fi
+
+echo "Installing dicedb binary to $DB_BIN"
+# curl -L "url to the latest release" -o dicedb
+sudo cp ./dice-server $DB_BIN
+sudo chmod 755 $DB_BIN
+
+# Creating systemd service
+if [ ! -f "$DB_SERVICE_FILE" ]; then
+    echo "Creating systemd service file: $DB_SERVICE_FILE"
+    sudo tee $DB_SERVICE_FILE > /dev/null <<EOL
+[Unit]
+Description=dicedb Service
+After=network.target
+
+[Service]
+User=$DB_USER
+Group=$DB_GROUP
+ExecStart=$DB_BIN
+Restart=on-failure
+
+[Install]
+WantedBy=multi-user.target
+EOL
+
+    sudo systemctl daemon-reload
```

Comment:
Wouldn't work on Darwin/MacOS

Reply:
Designed for linux user so not cross platform compatibility yet